tests/main_test.py

```diff
@@ -10,12 +10,68 @@
 from detect_secrets import VERSION
 from detect_secrets.core import audit as audit_module
 from detect_secrets.main import main
+from detect_secrets.plugins.common.util import import_plugins
 from testing.factories import secrets_collection_factory
 from testing.mocks import Any
 from testing.mocks import mock_printer
 from testing.util import uncolor
 
 
+def get_list_of_plugins(include=None, exclude=None):
```

Comment:
Super nit: One might expect `include` to mean 'only include', maybe `customized` and `excluded` might be more intuitive. Alternatively maybe a docstring saying `include` is only necessary for plugins w/out the defaults.